pom.xml

```diff
@@ -585,12 +585,27 @@
       </dependency>
       <dependency>
         <groupId>org.elasticsearch.plugin</groupId>
+        <artifactId>ingest-attachment</artifactId>
+        <version>${org.elasticsearch.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.elasticsearch.plugin</groupId>
         <artifactId>mapper-attachments</artifactId>
         <version>${org.elasticsearch.version}</version>
       </dependency>
       <dependency>
+        <groupId>org.codelibs.elasticsearch.module</groupId>
```

Comment:
I don't think so, as you can see the groupId is different from the official elasticsearch groupId (`org.codelibs.elasticsearch.module` instead of `org.elasticsearch.module`). This library is distributed by CodeLibs project https://github.com/codelibs which has no official relation with Elastic AFAIK. 

As you can see on the Maven Central, the official groupId for elasticsearch module is `org.elasticsearch.module` http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.elasticsearch.module%22

Since ES 5.0, Elastic doesn't upload anymore Maven artifacts on Maven Central, except for the distribution ZIP:
http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.elasticsearch.distribution.zip%22%20AND%20a%3A%22elasticsearch%22

So IMO, we have to do the same thing for all ES dependencies (upload artifacts in our Nexus).